lib/api/controllers/account_controller.ex

```diff
@@ -1,18 +1,79 @@
 defmodule NabooAPI.AccountController do
   use Phoenix.Controller, namespace: NabooAPI, root: "lib/api"
+  use PhoenixSwagger
 
   alias Naboo.Accounts
   alias Naboo.Accounts.Account
   alias NabooAPI.AccountView
   alias NabooAPI.Views.Errors
 
+  swagger_path(:index) do
+    get("api/account")
+    summary("Lists users")
+    description("Lists all users in the database")
+    produces("application/json")
+    deprecated(false)
+
+    response(200, "index.json", %{},
+      example: %{
+        accounts: [
+          %{
+            email: "test@test.com",
```

Comment:
As you can see [here](https://github.com/ARKultur/naboo/blob/dev/lib/naboo/accounts/account.ex#L5), we do not serialize `encrypted_password` or `password`